### DISCUSSION TITLE
v0.23.0-rc8 - Error rendering email template: open email-templates/base.html: no such file or directory

### DISCUSSION THREAD

**Discussion by MichaelSowah** [pocketbase/pocketbase#5781] [Q&A] [answered]
This is my structure 

email-templates/
    views/
        base.html
        verify-email.html.
    main.go

and this is my code
`var registry = template.NewRegistry()

	html, err := registry.LoadFiles(
		"email-templates/base.html",
		"email-templates/verify-email.html",
	).Render(map[string]interface{}{
		"code": OTP
	})
	if err != nil {
		log.Printf("Error rendering email template: %v", err)
		// return err
	}
	message := &mailer.Message{
		From: mail.Address{
			Address: app.Settings().Meta.SenderAddress,
			Name:    app.Settings().Meta.SenderName,
		},
		To:      []mail.Address{{Address: Recipient}},
		Subject: "Verify your email address",
		HTML:    html,
	}

	event := new(core.MailerRecordEvent)
	event.Mailer = mailClient
	event.Message = message

	return event.Mailer.Send(event.Message)
`
then when I GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build
and deploy to my cloud and run, and try triggering the email
I keep getting this error Error rendering email template: open email-templates/base.html: no such file or directory. The email was sent but had no content.

Kindly assist

**Comment by ganigeorgiev** [comment, marked as answer]
There are no changes to the templates rendering in v0.23.0-rc.

The error indicates that the template files are missing. Make sure that they exist on the server were the binary is deployed.

Note that the html templates and other assets are not embedded automatically in the final executable and if you want that you need to use the std [`//go:embed` directive](https://pkg.go.dev/embed) (_for example that embeds html templates you can check the code in one of my other open source projects - [presentator/presentator/mails.go#L22](https://github.com/presentator/presentator/blob/master/mails.go#L22)_).

**Comment by MichaelSowah** [comment]
Thank you @ganigeorgiev for the explanation

looking at your explanation, I was not [//go:embed directive](https://pkg.go.dev/embed), so that's why its throwing that error.

While waiting for a response to this issue, I moved my HTML template into an emails.go file  and used  template.New implementation that you used in pocketbase and that worked out.

I hope the template.New rendering right though?
